src/activity-window-port.js

```diff
@@ -173,15 +173,45 @@ export class ActivityWindowPort {
     }
 
     // Open the window.
+    let targetWin;
+    let openTarget = this.openTarget_;
+
+    // IE often returns `null` from the `window.open` API for a cross-origin
+    // target URL, making it impossible to set up messaging. The best path is
+    // to first open a blank window and then replace its location to the target
+    // url. This, however, needs to be done very carefully - if the browser
+    // instead forces redirect to "about:blank", the page's JS context will be
+    // destroy and it may fail to change the location to the target URL.
+    // As such, this option is only currently implemented for IE.
+    if (openTarget != '_top') {
+      const nav = this.win_.navigator;
+      const openViaBlank = /MSIE/i.test(nav && nav.userAgent);
```

Comment:
leave a comment here that MSIE represents IE?

Reply:
There's mentioning of it two lines above. Do you think another one needed here as well?

Reply:
SG. Added the comment. PTAL.